### DISCUSSION TITLE
Users can't select a model

### DISCUSSION THREAD

**Discussion by BoykaCheng** [open-webui/open-webui#7325] [General]
I started it with python, the permission is the user's account, and I can't select the model in the interface, but the account with the permission of the administrator can, ask for help

**Comment by HansonHA** [comment]
My situation is same to you, I started OI without Docker. My environment is win10 + miniconda + Python. Are there any ideas to fix this problem? I need a normal user without administration right to link models.

**Comment by cyhyraethz** [comment]
I'm runnning OI with docker compose and having the same issue.

**Comment by 0xkeenz** [comment]
I also encountered this problem after running OI in docker

**Comment by pmelendez** [comment]
This was a bit disorienting for me as well, but if you go to the admin settings and edit the model, you make them "public", which means they are available to all users.

**Reply by 0xkeenz** [reply]
Cool!

**Comment by derek-assurity** [comment]
From version `v0.4.8` there is a variable to control this behaviour. Set `BYPASS_MODEL_ACCESS_CONTROL` to `true` and all users will have access to all models.

**Reply by PeterAIFactory** [reply]
It works! Nice job bro👍

**Reply by GuokaiLiu** [reply]
Thanks for the share! It works. : )

**Comment by flefevre** [comment]
Perhaps you could close this discussion since it is resolved.
